stories/Configure.stories.js

```diff
@@ -24,4 +24,15 @@ storiesOf('Configure', module)
         this.hitsPerPage = this.hitsPerPage === 1 ? 5 : 1;
       },
     },
+  }))
+  .add('inline toggler', () => ({
+    template: `
+      <ais-configure :hitsPerPage="1">
+        <template slot-scope="{ searchParameters, refine }">
+          <pre>{{JSON.stringify(searchParameters, null, 2)}}</pre>
+          <button @click="refine({hitsPerPage: searchParameters.hitsPerPage === 1 ? 5 : 1})">
```

Comment:
toggling this will make us request with a whole helper.state, something like: 


```
curl 'https://latency-dsn.algolia.net/1/indexes/*/queries?x-algolia-agent=Algolia%20for%20vanilla%20JavaScript%20(lite)%203.27.0%3Binstantsearch.js%202.7.3%3BJS%20Helper%202.26.0&x-algolia-application-id=latency&x-algolia-api-key=6be0576ff61c053d5f9a3225e2a90f76' \
-XPOST \
-H 'Host: latency-dsn.algolia.net' \
-H 'Referer: https://deploy-preview-439--vue-instantsearch.netlify.com/stories/iframe.html?selectedKind=Configure&selectedStory=inline%20toggler' \
-H 'Accept: application/json' \
-H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.2 Safari/605.1.15' \
-H 'Accept-Language: en-us' \
-H 'Accept-Encoding: br, gzip, deflate' \
-H 'DNT: 1' \
-H 'Content-Length: 20714' \
-H 'Origin: https://deploy-preview-439--vue-instantsearch.netlify.com' \
-H 'Content-Type: application/x-www-form-urlencoded' \
-H 'Connection: keep-alive' \
--data $'{"requests":[{"indexName":"instant_search","params":"query=&hitsPerPage=5&page=0&clearRefinements=function%20clearRefinements(attribute)%7Bvar%20clear%3DRefinementList.clearRefinement%2Cpatch%3D%7BnumericRefinements%3Athis._clearNumericRefinements(attribute)%2CfacetsRefinements%3Aclear(this.facetsRefinements%2Cattribute%2C%22conjunctiveFacet%22)%2CfacetsExcludes%3Aclear(this.facetsExcludes%2Cattribute%2C%22exclude%22)%2CdisjunctiveFacetsRefinements%3Aclear(this.disjunctiveFacetsRefinements%2Cattribute%2C%22disjunctiveFacet%22)%2ChierarchicalFacetsRefinements%3Aclear(this.hierarchicalFacetsRefinements%2Cattribute%2C%22hierarchicalFacet%22)%7D%3Breturn%20patch.numericRefinements%3D%3D%3Dthis.numericRefinements%26%26patch.facetsRefinements%3D%3D%3Dthis.facetsRefinements%26%26patch.facetsExcludes%3D%3D%3Dthis.facetsExcludes%26%26patch.disjunctiveFacetsRefinements%3D%3D%3Dthis.disjunctiveFacetsRefinements%26%26patch.hierarchicalFacetsRefinements%3D%3D%3Dthis.hierarchicalFacetsRefinements%3Fthis%3Athis.setQueryParameters(patch)%7D&clearTags=function%20clearTags()%7Breturn%20void%200%3D%3D%3Dthis.tagFilters%26%260%3D%3D%3Dthis.tagRefinements.length%3Fthis%3Athis.setQueryParameters(%7BtagFilters%3Avoid%200%2CtagRefinements%3A%5B%5D%7D)%7D&setIndex=function%20setIndex(index)%7Breturn%20index%3D%3D%3Dthis.index%3Fthis%3Athis.setQueryParameters(%7Bindex%3Aindex%7D)%7D&setQuery=function%20setQuery(newQuery)%7Breturn%20newQuery%3D%3D%3Dthis.query%3Fthis%3Athis.setQueryParameters(%7Bquery%3AnewQuery%7D)%7D&setPage=function%20setPage(newPage)%7Breturn%20newPage%3D%3D%3Dthis.page%3Fthis%3Athis.setQueryParameters(%7Bpage%3AnewPage%7D)%7D&setFacets=function%20setFacets(facets)%7Breturn%20this.setQueryParameters(%7Bfacets%3Afacets%7D)%7D&setDisjunctiveFacets=function%20setDisjunctiveFacets(facets)%7Breturn%20this.setQueryParameters(%7BdisjunctiveFacets%3Afacets%7D)%7D&setHitsPerPage=function%20setHitsPerPage(n)%7Breturn%20this.hitsPerPage%3D%3D%3Dn%3Fthis%3Athis.setQueryParameters(%7BhitsPerPage%3An%7D)%7D&setTypoTolerance=function%20setTypoTolerance(typoTolerance)%7Breturn%20this.typoTolerance%3D%3D%3DtypoTolerance%3Fthis%3Athis.setQueryParameters(%7BtypoTolerance%3AtypoTolerance%7D)%7D&addNumericRefinement=function%20(attribute%2Coperator%2Cv)%7Bvar%20value%3DvalToNumber(v)%3Bif(this.isNumericRefined(attribute%2Coperator%2Cvalue))return%20this%3Bvar%20mod%3Dmerge(%7B%7D%2Cthis.numericRefinements)%3Breturn%20mod%5Battribute%5D%3Dmerge(%7B%7D%2Cmod%5Battribute%5D)%2Cmod%5Battribute%5D%5Boperator%5D%3F(mod%5Battribute%5D%5Boperator%5D%3Dmod%5Battribute%5D%5Boperator%5D.slice()%2Cmod%5Battribute%5D%5Boperator%5D.push(value))%3Amod%5Battribute%5D%5Boperator%5D%3D%5Bvalue%5D%2Cthis.setQueryParameters(%7BnumericRefinements%3Amod%7D)%7D&getConjunctiveRefinements=function%20(facetName)%7Bif(\041this.isConjunctiveFacet(facetName))throw%20new%20Error(facetName%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20this.facetsRefinements%5BfacetName%5D%7C%7C%5B%5D%7D&getDisjunctiveRefinements=function%20(facetName)%7Bif(\041this.isDisjunctiveFacet(facetName))throw%20new%20Error(facetName%2B%22%20is%20not%20defined%20in%20the%20disjunctiveFacets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20this.disjunctiveFacetsRefinements%5BfacetName%5D%7C%7C%5B%5D%7D&getHierarchicalRefinement=function%20(facetName)%7Breturn%20this.hierarchicalFacetsRefinements%5BfacetName%5D%7C%7C%5B%5D%7D&getExcludeRefinements=function%20(facetName)%7Bif(\041this.isConjunctiveFacet(facetName))throw%20new%20Error(facetName%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20this.facetsExcludes%5BfacetName%5D%7C%7C%5B%5D%7D&removeNumericRefinement=function%20(attribute%2Coperator%2CparamValue)%7Bif(void%200\041%3D%3DparamValue)%7Bvar%20paramValueAsNumber%3DvalToNumber(paramValue)%3Breturn%20this.isNumericRefined(attribute%2Coperator%2CparamValueAsNumber)%3Fthis.setQueryParameters(%7BnumericRefinements%3Athis._clearNumericRefinements(function(value%2Ckey)%7Breturn%20key%3D%3D%3Dattribute%26%26value.op%3D%3D%3Doperator%26%26isEqual(value.val%2CparamValueAsNumber)%7D)%7D)%3Athis%7Dreturn%20void%200\041%3D%3Doperator%3Fthis.isNumericRefined(attribute%2Coperator)%3Fthis.setQueryParameters(%7BnumericRefinements%3Athis._clearNumericRefinements(function(value%2Ckey)%7Breturn%20key%3D%3D%3Dattribute%26%26value.op%3D%3D%3Doperator%7D)%7D)%3Athis%3Athis.isNumericRefined(attribute)%3Fthis.setQueryParameters(%7BnumericRefinements%3Athis._clearNumericRefinements(function(value%2Ckey)%7Breturn%20key%3D%3D%3Dattribute%7D)%7D)%3Athis%7D&getNumericRefinements=function%20(facetName)%7Breturn%20this.numericRefinements%5BfacetName%5D%7C%7C%7B%7D%7D&getNumericRefinement=function%20(attribute%2Coperator)%7Breturn%20this.numericRefinements%5Battribute%5D%26%26this.numericRefinements%5Battribute%5D%5Boperator%5D%7D&_clearNumericRefinements=function%20_clearNumericRefinements(attribute)%7Bif(isUndefined(attribute))return%20isEmpty(this.numericRefinements)%3Fthis.numericRefinements%3A%7B%7D%3Bif(isString(attribute))return%20isEmpty(this.numericRefinements%5Battribute%5D)%3Fthis.numericRefinements%3Aomit(this.numericRefinements%2Cattribute)%3Bif(isFunction(attribute))%7Bvar%20hasChanged%3D\0411%2CnewNumericRefinements%3Dreduce(this.numericRefinements%2Cfunction(memo%2Coperators%2Ckey)%7Bvar%20operatorList%3D%7B%7D%3Breturn%20forEach(operators%2Cfunction(values%2Coperator)%7Bvar%20outValues%3D%5B%5D%3BforEach(values%2Cfunction(value)%7Battribute(%7Bval%3Avalue%2Cop%3Aoperator%7D%2Ckey%2C%22numeric%22)%7C%7CoutValues.push(value)%7D)%2CisEmpty(outValues)%3FhasChanged%3D\0410%3A(outValues.length\041%3D%3Dvalues.length%26%26(hasChanged%3D\0410)%2CoperatorList%5Boperator%5D%3DoutValues)%7D)%2CisEmpty(operatorList)%7C%7C(memo%5Bkey%5D%3DoperatorList)%2Cmemo%7D%2C%7B%7D)%3Breturn%20hasChanged%3FnewNumericRefinements%3Athis.numericRefinements%7D%7D&addFacet=function%20addFacet(facet)%7Breturn%20this.isConjunctiveFacet(facet)%3Fthis%3Athis.setQueryParameters(%7Bfacets%3Athis.facets.concat(%5Bfacet%5D)%7D)%7D&addDisjunctiveFacet=function%20addDisjunctiveFacet(facet)%7Breturn%20this.isDisjunctiveFacet(facet)%3Fthis%3Athis.setQueryParameters(%7BdisjunctiveFacets%3Athis.disjunctiveFacets.concat(%5Bfacet%5D)%7D)%7D&addHierarchicalFacet=function%20addHierarchicalFacet(hierarchicalFacet)%7Bif(this.isHierarchicalFacet(hierarchicalFacet.name))throw%20new%20Error(%22Cannot%20declare%20two%20hierarchical%20facets%20with%20the%20same%20name%3A%20%60%22%2BhierarchicalFacet.name%2B%22%60%22)%3Breturn%20this.setQueryParameters(%7BhierarchicalFacets%3Athis.hierarchicalFacets.concat(%5BhierarchicalFacet%5D)%7D)%7D&addFacetRefinement=function%20addFacetRefinement(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.facetsRefinements%2Cfacet%2Cvalue)%3Fthis%3Athis.setQueryParameters(%7BfacetsRefinements%3ARefinementList.addRefinement(this.facetsRefinements%2Cfacet%2Cvalue)%7D)%7D&addExcludeRefinement=function%20addExcludeRefinement(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.facetsExcludes%2Cfacet%2Cvalue)%3Fthis%3Athis.setQueryParameters(%7BfacetsExcludes%3ARefinementList.addRefinement(this.facetsExcludes%2Cfacet%2Cvalue)%7D)%7D&addDisjunctiveFacetRefinement=function%20addDisjunctiveFacetRefinement(facet%2Cvalue)%7Bif(\041this.isDisjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20disjunctiveFacets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.disjunctiveFacetsRefinements%2Cfacet%2Cvalue)%3Fthis%3Athis.setQueryParameters(%7BdisjunctiveFacetsRefinements%3ARefinementList.addRefinement(this.disjunctiveFacetsRefinements%2Cfacet%2Cvalue)%7D)%7D&addTagRefinement=function%20addTagRefinement(tag)%7Bif(this.isTagRefined(tag))return%20this%3Bvar%20modification%3D%7BtagRefinements%3Athis.tagRefinements.concat(tag)%7D%3Breturn%20this.setQueryParameters(modification)%7D&removeFacet=function%20removeFacet(facet)%7Breturn%20this.isConjunctiveFacet(facet)%3Fthis.clearRefinements(facet).setQueryParameters(%7Bfacets%3Afilter(this.facets%2Cfunction(f)%7Breturn%20f\041%3D%3Dfacet%7D)%7D)%3Athis%7D&removeDisjunctiveFacet=function%20removeDisjunctiveFacet(facet)%7Breturn%20this.isDisjunctiveFacet(facet)%3Fthis.clearRefinements(facet).setQueryParameters(%7BdisjunctiveFacets%3Afilter(this.disjunctiveFacets%2Cfunction(f)%7Breturn%20f\041%3D%3Dfacet%7D)%7D)%3Athis%7D&removeHierarchicalFacet=function%20removeHierarchicalFacet(facet)%7Breturn%20this.isHierarchicalFacet(facet)%3Fthis.clearRefinements(facet).setQueryParameters(%7BhierarchicalFacets%3Afilter(this.hierarchicalFacets%2Cfunction(f)%7Breturn%20f.name\041%3D%3Dfacet%7D)%7D)%3Athis%7D&removeFacetRefinement=function%20removeFacetRefinement(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.facetsRefinements%2Cfacet%2Cvalue)%3Fthis.setQueryParameters(%7BfacetsRefinements%3ARefinementList.removeRefinement(this.facetsRefinements%2Cfacet%2Cvalue)%7D)%3Athis%7D&removeExcludeRefinement=function%20removeExcludeRefinement(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.facetsExcludes%2Cfacet%2Cvalue)%3Fthis.setQueryParameters(%7BfacetsExcludes%3ARefinementList.removeRefinement(this.facetsExcludes%2Cfacet%2Cvalue)%7D)%3Athis%7D&removeDisjunctiveFacetRefinement=function%20removeDisjunctiveFacetRefinement(facet%2Cvalue)%7Bif(\041this.isDisjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20disjunctiveFacets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.disjunctiveFacetsRefinements%2Cfacet%2Cvalue)%3Fthis.setQueryParameters(%7BdisjunctiveFacetsRefinements%3ARefinementList.removeRefinement(this.disjunctiveFacetsRefinements%2Cfacet%2Cvalue)%7D)%3Athis%7D&removeTagRefinement=function%20removeTagRefinement(tag)%7Bif(\041this.isTagRefined(tag))return%20this%3Bvar%20modification%3D%7BtagRefinements%3Afilter(this.tagRefinements%2Cfunction(t)%7Breturn%20t\041%3D%3Dtag%7D)%7D%3Breturn%20this.setQueryParameters(modification)%7D&toggleRefinement=function%20toggleRefinement(facet%2Cvalue)%7Breturn%20this.toggleFacetRefinement(facet%2Cvalue)%7D&toggleFacetRefinement=function%20toggleFacetRefinement(facet%2Cvalue)%7Bif(this.isHierarchicalFacet(facet))return%20this.toggleHierarchicalFacetRefinement(facet%2Cvalue)%3Bif(this.isConjunctiveFacet(facet))return%20this.toggleConjunctiveFacetRefinement(facet%2Cvalue)%3Bif(this.isDisjunctiveFacet(facet))return%20this.toggleDisjunctiveFacetRefinement(facet%2Cvalue)%3Bthrow%20new%20Error(%22Cannot%20refine%20the%20undeclared%20facet%20%22%2Bfacet%2B%22%3B%20it%20should%20be%20added%20to%20the%20helper%20options%20facets%2C%20disjunctiveFacets%20or%20hierarchicalFacets%22)%7D&toggleConjunctiveFacetRefinement=function%20toggleConjunctiveFacetRefinement(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20this.setQueryParameters(%7BfacetsRefinements%3ARefinementList.toggleRefinement(this.facetsRefinements%2Cfacet%2Cvalue)%7D)%7D&toggleExcludeFacetRefinement=function%20toggleExcludeFacetRefinement(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20this.setQueryParameters(%7BfacetsExcludes%3ARefinementList.toggleRefinement(this.facetsExcludes%2Cfacet%2Cvalue)%7D)%7D&toggleDisjunctiveFacetRefinement=function%20toggleDisjunctiveFacetRefinement(facet%2Cvalue)%7Bif(\041this.isDisjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20disjunctiveFacets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20this.setQueryParameters(%7BdisjunctiveFacetsRefinements%3ARefinementList.toggleRefinement(this.disjunctiveFacetsRefinements%2Cfacet%2Cvalue)%7D)%7D&toggleHierarchicalFacetRefinement=function%20toggleHierarchicalFacetRefinement(facet%2Cvalue)%7Bif(\041this.isHierarchicalFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20hierarchicalFacets%20attribute%20of%20the%20helper%20configuration%22)%3Bvar%20separator%3Dthis._getHierarchicalFacetSeparator(this.getHierarchicalFacetByName(facet))%2Cmod%3D%7B%7D%3Breturn%20void%200\041%3D%3Dthis.hierarchicalFacetsRefinements%5Bfacet%5D%26%26this.hierarchicalFacetsRefinements%5Bfacet%5D.length%3E0%26%26(this.hierarchicalFacetsRefinements%5Bfacet%5D%5B0%5D%3D%3D%3Dvalue%7C%7C0%3D%3D%3Dthis.hierarchicalFacetsRefinements%5Bfacet%5D%5B0%5D.indexOf(value%2Bseparator))%3F-1%3D%3D%3Dvalue.indexOf(separator)%3Fmod%5Bfacet%5D%3D%5B%5D%3Amod%5Bfacet%5D%3D%5Bvalue.slice(0%2Cvalue.lastIndexOf(separator))%5D%3Amod%5Bfacet%5D%3D%5Bvalue%5D%2Cthis.setQueryParameters(%7BhierarchicalFacetsRefinements%3Adefaults(%7B%7D%2Cmod%2Cthis.hierarchicalFacetsRefinements)%7D)%7D&addHierarchicalFacetRefinement=function%20(facet%2Cpath)%7Bif(this.isHierarchicalFacetRefined(facet))throw%20new%20Error(facet%2B%22%20is%20already%20refined.%22)%3Bvar%20mod%3D%7B%7D%3Breturn%20mod%5Bfacet%5D%3D%5Bpath%5D%2Cthis.setQueryParameters(%7BhierarchicalFacetsRefinements%3Adefaults(%7B%7D%2Cmod%2Cthis.hierarchicalFacetsRefinements)%7D)%7D&removeHierarchicalFacetRefinement=function%20(facet)%7Bif(\041this.isHierarchicalFacetRefined(facet))throw%20new%20Error(facet%2B%22%20is%20not%20refined.%22)%3Bvar%20mod%3D%7B%7D%3Breturn%20mod%5Bfacet%5D%3D%5B%5D%2Cthis.setQueryParameters(%7BhierarchicalFacetsRefinements%3Adefaults(%7B%7D%2Cmod%2Cthis.hierarchicalFacetsRefinements)%7D)%7D&toggleTagRefinement=function%20toggleTagRefinement(tag)%7Breturn%20this.isTagRefined(tag)%3Fthis.removeTagRefinement(tag)%3Athis.addTagRefinement(tag)%7D&isDisjunctiveFacet=function%20(facet)%7Breturn%20indexOf(this.disjunctiveFacets%2Cfacet)%3E-1%7D&isHierarchicalFacet=function%20(facetName)%7Breturn%20void%200\041%3D%3Dthis.getHierarchicalFacetByName(facetName)%7D&isConjunctiveFacet=function%20(facet)%7Breturn%20indexOf(this.facets%2Cfacet)%3E-1%7D&isFacetRefined=function%20isFacetRefined(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.facetsRefinements%2Cfacet%2Cvalue)%7D&isExcludeRefined=function%20isExcludeRefined(facet%2Cvalue)%7Bif(\041this.isConjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20facets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.facetsExcludes%2Cfacet%2Cvalue)%7D&isDisjunctiveFacetRefined=function%20isDisjunctiveFacetRefined(facet%2Cvalue)%7Bif(\041this.isDisjunctiveFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20disjunctiveFacets%20attribute%20of%20the%20helper%20configuration%22)%3Breturn%20RefinementList.isRefined(this.disjunctiveFacetsRefinements%2Cfacet%2Cvalue)%7D&isHierarchicalFacetRefined=function%20isHierarchicalFacetRefined(facet%2Cvalue)%7Bif(\041this.isHierarchicalFacet(facet))throw%20new%20Error(facet%2B%22%20is%20not%20defined%20in%20the%20hierarchicalFacets%20attribute%20of%20the%20helper%20configuration%22)%3Bvar%20refinements%3Dthis.getHierarchicalRefinement(facet)%3Breturn%20value%3F-1\041%3D%3DindexOf(refinements%2Cvalue)%3Arefinements.length%3E0%7D&isNumericRefined=function%20isNumericRefined(attribute%2Coperator%2Cvalue)%7Bif(isUndefined(value)%26%26isUndefined(operator))return\041\041this.numericRefinements%5Battribute%5D%3Bvar%20isOperatorDefined%3Dthis.numericRefinements%5Battribute%5D%26%26\041isUndefined(this.numericRefinements%5Battribute%5D%5Boperator%5D)%3Bif(isUndefined(value)%7C%7C\041isOperatorDefined)return%20isOperatorDefined%3Bvar%20parsedValue%3DvalToNumber(value)%2CisAttributeValueDefined%3D\041isUndefined(function%20findArray(array%2CsearchedValue)%7Breturn%20find(array%2Cfunction(currentValue)%7Breturn%20isEqual(currentValue%2CsearchedValue)%7D)%7D(this.numericRefinements%5Battribute%5D%5Boperator%5D%2CparsedValue))%3Breturn%20isOperatorDefined%26%26isAttributeValueDefined%7D&isTagRefined=function%20isTagRefined(tag)%7Breturn-1\041%3D%3DindexOf(this.tagRefinements%2Ctag)%7D&getRefinedDisjunctiveFacets=function%20getRefinedDisjunctiveFacets()%7Bvar%20disjunctiveNumericRefinedFacets%3Dintersection(keys(this.numericRefinements)%2Cthis.disjunctiveFacets)%3Breturn%20keys(this.disjunctiveFacetsRefinements).concat(disjunctiveNumericRefinedFacets).concat(this.getRefinedHierarchicalFacets())%7D&getRefinedHierarchicalFacets=function%20getRefinedHierarchicalFacets()%7Breturn%20intersection(map(this.hierarchicalFacets%2C%22name%22)%2Ckeys(this.hierarchicalFacetsRefinements))%7D&getUnrefinedDisjunctiveFacets=function%20()%7Bvar%20refinedFacets%3Dthis.getRefinedDisjunctiveFacets()%3Breturn%20filter(this.disjunctiveFacets%2Cfunction(f)%7Breturn-1%3D%3D%3DindexOf(refinedFacets%2Cf)%7D)%7D&managedParameters=%5B%22index%22%2C%22facets%22%2C%22disjunctiveFacets%22%2C%22facetsRefinements%22%2C%22facetsExcludes%22%2C%22disjunctiveFacetsRefinements%22%2C%22numericRefinements%22%2C%22tagRefinements%22%2C%22hierarchicalFacets%22%2C%22hierarchicalFacetsRefinements%22%5D&getQueryParams=function%20getQueryParams()%7Bvar%20managedParameters%3Dthis.managedParameters%2CqueryParams%3D%7B%7D%3Breturn%20forOwn(this%2Cfunction(paramValue%2CparamName)%7B-1%3D%3D%3DindexOf(managedParameters%2CparamName)%26%26void%200\041%3D%3DparamValue%26%26(queryParams%5BparamName%5D%3DparamValue)%7D)%2CqueryParams%7D&getQueryParameter=function%20getQueryParameter(paramName)%7Bif(\041this.hasOwnProperty(paramName))throw%20new%20Error(%22Parameter%20\'%22%2BparamName%2B%22\'%20is%20not%20an%20attribute%20of%20SearchParameters%20(http%3A%2F%2Falgolia.github.io%2Falgoliasearch-helper-js%2Fdocs%2FSearchParameters.html)%22)%3Breturn%20this%5BparamName%5D%7D&setQueryParameter=function%20setParameter(parameter%2Cvalue)%7Bif(this%5Bparameter%5D%3D%3D%3Dvalue)return%20this%3Bvar%20modification%3D%7B%7D%3Breturn%20modification%5Bparameter%5D%3Dvalue%2Cthis.setQueryParameters(modification)%7D&setQueryParameters=function%20setQueryParameters(params)%7Bif(\041params)return%20this%3Bvar%20error%3DSearchParameters.validate(this%2Cparams)%3Bif(error)throw%20error%3Bvar%20parsedParams%3DSearchParameters._parseNumbers(params)%3Breturn%20this.mutateMe(function%20mergeWith(newInstance)%7Bvar%20ks%3Dkeys(params)%3Breturn%20forEach(ks%2Cfunction(k)%7BnewInstance%5Bk%5D%3DparsedParams%5Bk%5D%7D)%2CnewInstance%7D)%7D&filter=function%20(filters)%7Breturn%20filterState(this%2Cfilters)%7D&mutateMe=function%20mutateMe(fn)%7Bvar%20newState%3Dnew%20this.constructor(this)%3Breturn%20fn(newState%2Cthis)%2CnewState%7D&_getHierarchicalFacetSortBy=function%20(hierarchicalFacet)%7Breturn%20hierarchicalFacet.sortBy%7C%7C%5B%22isRefined%3Adesc%22%2C%22name%3Aasc%22%5D%7D&_getHierarchicalFacetSeparator=function%20(hierarchicalFacet)%7Breturn%20hierarchicalFacet.separator%7C%7C%22%20%3E%20%22%7D&_getHierarchicalRootPath=function%20(hierarchicalFacet)%7Breturn%20hierarchicalFacet.rootPath%7C%7Cnull%7D&_getHierarchicalShowParentLevel=function%20(hierarchicalFacet)%7Breturn%22boolean%22\041%3Dtypeof%20hierarchicalFacet.showParentLevel%7C%7ChierarchicalFacet.showParentLevel%7D&getHierarchicalFacetByName=function%20(hierarchicalFacetName)%7Breturn%20find(this.hierarchicalFacets%2C%7Bname%3AhierarchicalFacetName%7D)%7D&getHierarchicalFacetBreadcrumb=function%20(facetName)%7Bif(\041this.isHierarchicalFacet(facetName))throw%20new%20Error(%22Cannot%20get%20the%20breadcrumb%20of%20an%20unknown%20hierarchical%20facet%3A%20%60%22%2BfacetName%2B%22%60%22)%3Bvar%20refinement%3Dthis.getHierarchicalRefinement(facetName)%5B0%5D%3Bif(\041refinement)return%5B%5D%3Bvar%20separator%3Dthis._getHierarchicalFacetSeparator(this.getHierarchicalFacetByName(facetName))%2Cpath%3Drefinement.split(separator)%3Breturn%20map(path%2Ctrim)%7D&toString=function%20()%7Breturn%20JSON.stringify(this%2Cnull%2C2)%7D&facets=%5B%5D&tagFilters="}]}'
```

Reply:
test it here: https://deploy-preview-439--vue-instantsearch.netlify.com/stories/?selectedKind=Configure&selectedStory=inline%20toggler&full=0&addons=0&stories=1&panelRight=0

Reply:
this is fixed in https://github.com/algolia/instantsearch.js/pull/2945